static/app/views/starfish/views/spanSummary/sidebar.tsx

```diff
@@ -0,0 +1,235 @@
+
+import Duration from 'sentry/components/duration';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {formatPercentage} from 'sentry/utils/formatters';
+import Chart from 'sentry/views/starfish/components/chart';
+import {HOST} from 'sentry/views/starfish/modules/APIModule/APIModuleView';
+import {
+  getEndpointDetailSeriesQuery,
+  getEndpointDetailTableQuery,
+} from 'sentry/views/starfish/modules/APIModule/queries';
+import {zeroFillSeries} from 'sentry/views/starfish/utils/zeroFillSeries';
+import {getUniqueTransactionCountQuery} from 'sentry/views/starfish/views/spanSummary/queries';
+
+export default function Sidebar({description, transactionName}) {
+  const theme = useTheme();
+  const seriesQuery = getEndpointDetailSeriesQuery({
+    description,
+    transactionName,
+  });
+  const aggregatesQuery = getEndpointDetailTableQuery({
+    description,
+    transactionName,
+  });
+
+  // This is supposed to a metrics span query that fetches aggregate metric data
+  const {isLoading: _isLoadingSideBarAggregateData, data: aggregateData} = useQuery({
+    queryKey: [aggregatesQuery],
+    queryFn: () => fetch(`${HOST}/?query=${aggregatesQuery}`).then(res => res.json()),
+    retry: false,
+    initialData: [],
+  });
+
+  // Also a metrics span query that fetches series data
+  const {isLoading: isLoadingSeriesData, data: seriesData} = useQuery({
+    queryKey: [seriesQuery],
+    queryFn: () => fetch(`${HOST}/?query=${seriesQuery}`).then(res => res.json()),
+    retry: false,
+    initialData: [],
+  });
+
+  // This is a metrics request on transactions data!
+  // We're fetching the count of events on a specific transaction so we can
+  // calculate span frequency using metrics spans vs metrics transactions
+  const countUniqueQuery = getUniqueTransactionCountQuery(transactionName);
+  const {data: transactionData, isLoading: _isTransactionDataLoading} = useQuery({
+    queryKey: [countUniqueQuery],
+    queryFn: () =>
+      fetch(`/api/0/organizations/sentry/events/${countUniqueQuery}`).then(res =>
+        res.json()
+      ),
+    retry: false,
+    initialData: [],
+  });
+
+  const {
+    p50,
+    p95,
+    failure_rate,
+    count,
+    total_exclusive_time,
+    count_unique_transaction_id,
+  } = aggregateData[0] || {};
+
+  const [p50Series, p95Series, countSeries, _errorCountSeries, errorRateSeries] =
+    queryDataToChartData(seriesData).map(series =>
+      zeroFillSeries(series, moment.duration(12, 'hours'))
+    );
+
+  // One thing we're doing here is multiplying the count_unique_transaction_id by 100000.
+  // This is because we're using scraped spans data (which doesn't have nearly enough volume).
+  // So we multiply by 100000 to simulate a higher volume of spans, ie metrics spans data.
+  const spanFrequency =
+    count_unique_transaction_id && transactionData?.data?.[0]?.['count()']
+      ? (count_unique_transaction_id * 100000) / transactionData.data[0]['count()']
+      : 0;
```

Comment:
I think you'd be better off not doing this, and let the data look broken! That way it's consistently bad, and we won't overlook it

Reply:
Good point! I've removed this and added a note so the next person can understand whats going on here!

Reply:
Right on!